CHANGELOG.md

```diff
@@ -15,13 +15,19 @@ FEATURES:
 * **New build commands** `make test-env-init` and `make test-env-apply` can configure an empty vCD to run the test suite. See `TESTING.md` for details.
 * `resource/vcd_org_vdc` added Org VDC update and full state read - [GH-275]
 * `resource/vcd_org_vdc` added Org VDC metadata support - [GH-276]
+* `resource/vcd_snat` added ability to choose network name and type. [GH-282] 
+* `resource/vcd_dnat` added ability to choose network name and type. [GH-282]
 
 IMPROVEMENTS:
 * `resource/vcd_org_vdc`: Fix ignoring of resource guarantee values - [GH-265]
 * `resource/vcd_org_vdc`: Org VDC state ID changed from name to vCD ID - [GH-275]
 * Change resource handling to use locking mechanism when resource parallel handling is not supported by vCD. [GH-255] 
 * Fix issue when vApp is power cycled during member VM deletion. [GH-261]
 
+BUG FIXES:
+
+* `resource/vcd_dnat and resource/vcd_snat` - delete works 100% if you use network_name and type [GH-282]
```

Comment:
Maybe we need a bit more context here. For someone not knowing the problem it wouldn't make much sense.

Reply:
Changed